tests/album-create.test.js

```diff
@@ -0,0 +1,43 @@
+const { expect } = require('chai');
+const request = require('supertest');
+const db = require('../src/db');
+const app = require('../src/app');
+
+describe('create album', () => {
+    let artistId;
+    beforeEach(async () => {
+        const { status, body } = await request(app)
+          .post('/artists')
+          .send({ name: 'Kurt Cobain', genre: 'rock' });
+
+          expect(status).to.equal(201);
+          expect(body.name).to.equal('Kurt Cobain');
+          expect(body.genre).to.equal('rock');
+          artistId = body.id;
```

Comment:
To grab the artistId you will have to extract that artist data from the row variable and the access the id from there
```
          artist = rows[0]; //you will need to declare this variable near the declaration of artistId above the beforeEach block
```